community/kernel/src/test/java/org/neo4j/test/OnDemandJobScheduler.java

```diff
@@ -50,6 +51,12 @@ public JobHandle schedule( Group group, Runnable job )
     }
 
     @Override
+    public JobHandle schedule( Group group, Runnable job, Map<String,String> metadata )
```

Comment:
How about make `schedule( Group, Runnable )` to invoke this one with `NO_METADATA`?